ArduPlane/RC_Channel.cpp

```diff
@@ -27,13 +27,40 @@ bool RC_Channels_Plane::has_valid_input() const
     return true;
 }
 
+void RC_Channel_Plane::do_aux_function_change_mode(const Mode::Number number,
+                                                   const aux_switch_pos_t ch_flag)
+{
+    switch(ch_flag) {
+    case HIGH: {
+        // engage mode (if not possible we remain in current flight mode)
+        const bool success = plane.set_mode_by_number(number, MODE_REASON_TX_COMMAND);
+        if (plane.control_mode != &plane.mode_initializing) {
+            if (success) {
+                AP_Notify::events.user_mode_change = 1;
```

Comment:
Not new to this, but whatever clears this flag? Presumably it gets cleared somewhere, but I can't figure out where/how....

Reply:
> Not new to this, but whatever clears this flag? Presumably it gets cleared somewhere, but I can't figure out where/how....

https://github.com/ardupilot/ardupilot/blob/master/libraries/AP_Notify/AP_Notify.cpp#L330